utils/src/ooinstall/cli_installer.py

```diff
@@ -722,14 +722,30 @@ def upgrade(ctx):
         click.echo("No hosts defined in: %s" % oo_cfg.config_path)
         sys.exit(1)
 
-    # Update config to reflect the version we're targetting, we'll write
-    # to disk once ansible completes successfully, not before.
     old_variant = oo_cfg.settings['variant']
     old_version = oo_cfg.settings['variant_version']
```

Comment:
Instead of trusting the version in the config, would it be better to call out to the openshift_facts playbook and get the version from the host? Or maybe use a combination of both approaches?